ext/FluxnetSimulations.jl

```diff
@@ -0,0 +1,308 @@
+module FluxnetSimulations
+import ClimaUtilities.TimeVaryingInputs:
+    TimeVaryingInput, LinearInterpolation, PeriodicCalendar
+import ClimaUtilities.TimeManager: ITime, date
+using Thermodynamics
+using Dates
+using DelimitedFiles
+using Format
+using DocStringExtensions
+using Insolation
+import ClimaLand.Parameters as LP
+using ClimaLand
+export prescribed_forcing_fluxnet, set_fluxnet_ic!
+"""
+     prescribed_forcing_fluxnet(site_ID,
+                                lat,
+                                long,
+                                hour_offset_from_UTC,
+                                start_date,
+                                earth_param_set,
+                                FT;
+                                gustiness=1,
+                                c_co2 = TimeVaryingInput((t) -> 4.2e-4),
+                                time_interpolation_method = LinearInterpolation(PeriodicCalendar())) # not used??
+
+A helper function which constructs the `PrescribedAtmosphere` and `PrescribedRadiativeFluxes`
+from a file path pointing to the Fluxnet data in a csv file, the start date, latitude, longitude,
+the hour offset of the site from UTC (local_time + offset = time in UTC),
+and the earth_param_set.
+"""
+function prescribed_forcing_fluxnet(
+    site_ID,
+    lat,
+    long,
+    hour_offset_from_UTC,
+    atmos_h,
+    start_date, # in UTC
+    earth_param_set,
+    FT;
+    split_precip = true,
+    gustiness = 1,
+    c_co2 = TimeVaryingInput((t) -> 4.2e-4),
+    time_interpolation_method = LinearInterpolation(PeriodicCalendar()),
+)
+    fluxnet_csv_path = ClimaLand.Artifacts.experiment_fluxnet_data_path(site_ID)
+    driver_data = readdlm(fluxnet_csv_path, ',')
+    columns = driver_data[1, :]
+    labels = ("TA_F", "VPD_F", "PA_F", "P_F", "WS_F", "LW_IN_F", "SW_IN_F")
+    indices = [findfirst(columns .== label) for label in labels]
+
+    if any(indices .== nothing)
+        nothing_id = findall(indices .== nothing)
+        @error("$(labels[nothing_id]) is missing in the data, but required.")
+    end
+
+    index_map = Dict(zip(labels, indices))
```

Comment:
Can you add some comments for what's being done here? It's not very clear to me why we need this

Reply:
I can add the comments if we dont switch to DataFrames

Reply:
I added some comments